modules/graceful-shutdown.adoc

```diff
@@ -25,7 +25,7 @@ It is important to take an etcd backup before performing this procedure so that
 +
 [source,terminal]
 ----
-$ nodes=$(oc get nodes -o name)
+$ nodes=$(oc get nodes -o jsonpath='{range .items[*]}{.metadata.name}{end})
```

Comment:
you don't need range, the default is whitespace delimited.

```shell
nodes=$(oc get nodes -o wide -o jsonpath='{.items[*].metadata.name}')
```

Reply:
@rbbratta thanks!

Reply:
err, bad copy-paste on my part, the extra `-o wide` is not needed.

`nodes=$(oc get nodes -o jsonpath='{.items[*].metadata.name}')`